bamboo/integration_tests/performance_tests/test_tools.py

```diff
@@ -1,24 +1,25 @@
+import csv
 import os
 import re
 import sys
 import time
 import unittest
 
-def test_model(model_name):
-  return '--model=model_zoo/tests/model_%s.prototext' % model_name
+def model_test(model_name):
+  return '--model=../model_zoo/tests/model_%s.prototext' % model_name
 
 def model(model_name):
-  return '--model=model_zoo/models/%s/model_%s.prototext' % (model_name, model_name)
+  return '--model=../model_zoo/models/%s/model_%s.prototext' % (model_name, model_name)
 
 def reader(reader_name):
-  return '--reader=model_zoo/data_readers/data_reader_%s.prototext' % reader_name
+  return '--reader=../model_zoo/data_readers/data_reader_%s.prototext' % reader_name
 
 def optimizer(optimizer_name):
-  return '--optimizer=model_zoo/optimizers/opt_%s.prototext' % optimizer_name
+  return '--optimizer=../model_zoo/optimizers/opt_%s.prototext' % optimizer_name
 
 def run_lbann(model_name, reader_name, optimizer_name, output_file_name, executable, num_nodes=1, timeout=60, num_processes=2):
   if model_name == 'mnist_distributed_io':
-    m = test_model(model_name)
+    m = model_test(model_name)
```

Comment:
Add comment in the code: "Must not be named 'test_model' or else pytest will try to run it."